plugin/plugins/qq_auto_reply/doc/README.ja.md

```
@@ -0,0 +1,154 @@
+# QQ 自動返信プラグイン
+
+OneBot プロトコル経由で QQ に接続し、権限に応じたインテリジェントな自動返信を行うプラグインです。個人チャットとグループメッセージの両方に対応し、AI 会話機能も統合されています。
+
+## 起動とガイド
+
+1. OneBot 実装を用意してください（NapCat 推奨）。以下では NapCat を例にします。
+   開く:
+   ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)